streamtasks/src/main/java/com/leverx/task/entity/Task.java

```diff
@@ -0,0 +1,42 @@
+package com.leverx.task.entity;
+
+import com.leverx.task.entity.enums.TaskType;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+
+@RequiredArgsConstructor
+public class Task {
+    @Getter
```

Comment:
If every variable has its own getter why don't we mark the class with this annotation?